packages/components/src/panel/actions.native.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	TextControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import styles from './actions.scss';
+
+function PanelActions( { actions } ) {
+	return (
+		<View style={ styles.panelActionsContainer }>
+			{ actions.map( ( { label, onPress } ) => {
+				return (
+					<TextControl
```

Comment:
I missed the PR where this happened, but this doesn't make sense. [TextControl](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/text-control) is supposed to be a component that "let users enter and edit text":

![Unfilled and filled TextControl components](https://make.wordpress.org/design/files/2019/03/TextControl.png)

This looks a lot more like a `Button`

Reply:
you are absolutely right, there's a tech debt here we should be addressing, thanks for opening https://github.com/WordPress/gutenberg/issues/18018